prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh

```diff
@@ -196,7 +196,8 @@ metadata:
     component: knative-eventing-channel-kafka
     kyma-project.io/installation: ""
 stringData:
-  kafka.brokers: ${K8S_SECRET_BROKER}
+  kafka.brokers.hostname: ${K8S_SECRET_BROKER}
+  kafka.brokers.port: ${KAFKA_BROKER_PORT}
```

Comment:
i know you did not introduce this variable:
but: the value is hardcoded to 9093. Would it make sense to make this default to 9093 if not set, otherwise use the value specified in the environment?

Reply:
@k15r Thanks for the review.
I thought that the behavior you're asking for is the current implementation given https://github.com/kyma-project/test-infra/pull/2198/files#diff-ae7dbfa9acdca658c142a283090aa43cL61
If not, would you please elaborate with an example? Thanks!

Reply:
Is it possible to derive the port from the `primaryConnectionString`?

Reply:
The port is not in the primaryConnectionString.